hosting/src/store/index.ts

```diff
@@ -15,7 +16,13 @@ const store = createStore<PTVuexState>({
     currentUser: null,
     events: []
   },
-  getters: {},
+  getters: {
+    events: (state) => state.events.map(eventToEventUi),
+    eventBySlug: (state) => (slug: string) => {
+      const event = state.events.find(event => event.slug === slug)
+      return event ? eventToEventUi(event) : undefined
```

Comment:
How do you feel about a utility function in common: `findMap`
```ts
export const findMap<T, U>(iterable: Iterable<T>?, findFn: (T) => boolean, mapFn: (T) => U): U? {
  const findResult = iterable?.find(findFn)
  return findResult ? mapFn(findResult) : undefined
}
```
...which you could then call:
```ts
events: (state) => map(state.events, eventToEventUi), // lodash map
eventBySlug: (state) => findMap(state.events, event => event.slug === slug, eventToEventUi)
```